tests/mmptest/src/FrameworkLinksTests.cs

```diff
@@ -114,5 +114,21 @@ void AssertLinkedFrameworkStatus (string[] clangParts)
 
 			AssertFrameworkMinOSRespected (status);
 		}
+
+
+		[Test]
+		public void ProjectWithLinkToPrivateFramework_ShouldBuild ()
+		{
+			RunMMPTest (tmpDir => {
+
+				TI.UnifiedTestConfig test = new TI.UnifiedTestConfig (tmpDir) { TestDecl = @"
+	const string MobileDeviceLibrary = ""/System/Library/PrivateFrameworks/MobileDevice.framework/MobileDevice"";
+	[System.Runtime.InteropServices.DllImport (MobileDeviceLibrary)]
+	private static extern void TestMethod ();
+				" };
```

Comment:
If you have a native reference in your project that's a framework we have other code that copies it in and does the work to make it work.

This would just be p/invokes into frameworks not set as a native reference.